.github/CODEOWNERS

```diff
@@ -0,0 +1,40 @@
+# The purpose of this file is to trigger review requests when PRs touch
+# particular files. Those reviews are not mandatory, however it's often useful
+# to have an expert pinged who is interested in only one part of SciPy and
+# doesn't follow general development.
+#
+# Note that only GitHub handles (whether individuals or teams) with commit
+# rights should be added to this file.
+# See https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# for more details about how CODEOWNERS works.
+
+# Each line is a file pattern followed by one or more owners.
+
+.github/CODEOWNERS @rgommers @larsoner
+
+# Build related files
+pyproject.toml @rgommers
+setup.py @rgommers @larsoner
+
+# SciPy submodules
+scipy/stats/*qmc.* @tupui
+scipy/signal/* @larsoner
+scipy/linalg/* @larsoner
+scipy/sparse/* @perimosocordiae
```

Comment:
`scipy/spatial/*` @tylerjereddy  (perhaps also @pmla and @peterbell10 based on recent activity?)

Reply:
+1

Reply:
all of spatial @peterbell10, or only `spatial.distance`?

Reply:
Probably easier to do all of `spatial`. distance and KDTree which aren't neatly in their own folders.

Reply:
oh right, forgot about KDTree:)